### ISSUE TITLE
Cookie not sent through swagger ui

### ISSUE THREAD

**Issue by pawamoy** [tiangolo/fastapi#880]
Not sure where to open this issue. The Swagger-UI bug report template says:
> Issues on GitHub are only related to problems of Swagger-UI itself. We'll try
  to offer support here for your use case, but we can't offer help with projects
  that use Swagger-UI indirectly, like Springfox or swagger-node.
>
> Likewise, we can't accept bugs in the Swagger/OpenAPI specifications
  themselves, or anything that violates the specifications.

### Describe the bug

One of my routes expects a Cookie parameter. When I paste the cookie value in the Swagger form field (which is correctly typed as "cookie") and execute the query, then the cookie I receive is None (the default value).

### To Reproduce

1. Create a file with:

```python
import uvicorn
from fastapi import Depends, FastAPI, Cookie, HTTPException
from pydantic import BaseModel
from starlette.status import HTTP_401_UNAUTHORIZED

app = FastAPI()


class User(BaseModel):
    username: str


def validate_cookie(cookie: str):
    # fake get username by testing cookie against another service
    if cookie:
        return User(username="johnette")

    raise HTTPException(
        status_code=HTTP_401_UNAUTHORIZED,
        detail="Invalid cookie"
    )


async def get_current_user(cookie: str = Cookie(None)):
    user = validate_cookie(cookie)
    return user


@app.get("/someroute/")
def some_route(current_user: User = Depends(get_current_user)) -> str:
    return current_user.username


if __name__ == "__main__":
    uvicorn.run(app, host="0.0.0.0", port=8000)

```

3. Open the browser the Swagger docs and try the endpoint `/someroute/` by filling the cookie value.
4. It returns a 401 Unauthorized: Invalid cookie.
5. But I expected it to return `"johnette"`.

### Expected behavior

I expected the cookie to be sent and not getting None.
If I run the `curl`command Swagger UI is showing, it's working as expected (I get `"johnette"`).

### Screenshots

Through the web UI:
![swagger_bug](https://user-images.githubusercontent.com/3999221/72614130-bfaa2100-3931-11ea-80b5-e41a26485390.PNG)

With curl:
![swagger_bug_cli](https://user-images.githubusercontent.com/3999221/72614132-c33da800-3931-11ea-8a99-f3a0a013c336.PNG)

### Environment

- OS: Windows
- FastAPI Version 0.45.0
- Python version 3.6.10 :: Anaconda Inc.

**Comment by pawamoy** [comment]
Based on https://swagger.io/docs/specification/authentication/cookie-authentication/, it seems Cookie authentication is not possible anymore with Swagger UI "try it out".

Anyway, it's fine because I can use my login route to get the cookie in my browser session, and then let the cookie field empty.

**Comment by dmontagu** [comment]
Thanks for reporting back with the answer to this!

**Comment by tiangolo** [comment]
Yep, Cookie authentication is difficult to debug and modify while running on the web browser docs UI as the browsers have security restrictions for JS on the browser. Nevertheless, if you have an endpoint that sets a Cookie, and then you try to use it with other endpoints, it should work normally. What you can't do is provide a cookie value in the docs UI.

Anyway, thanks for reporting back and closing the issue :+1:

**Comment by yifeikong** [comment]
I think this behavior should be highlighted in the cookie parameter section in the docs, which will save me much time for googling this.
